### DISCUSSION TITLE
docs/community-resources/recommended-reading-list

### DISCUSSION THREAD

**Discussion by giscus** [OfferZen-Community/developer-mentoring#140] [Announcements]
# docs/community-resources/recommended-reading-list

Two-way feedback is a super important part of any mentoring relationship. It’s also an essential ingredient in career development work. That’s why Project Thrive participants make use of the continuous feedback loop to learn how to give and receive feedback, and to make sure everyone is on the same page and communication is honest, open and constructive. Here’s how it works.

https://www.developermentoring.guide/docs/community-resources/recommended-reading-list

**Comment by lasse-offerzen** [comment]
One other book that gets mentioned a lot is [The Programmer's Brain](https://www.amazon.com/Programmers-Brain-every-programmer-cognition/dp/1617298670) 🧠📚
